src/Tools/Console/Command/RunSqlCommand.php

```diff
@@ -75,23 +76,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         assert(is_string($sql));
 
-        $depth = $input->getOption('depth');
-
-        if (! is_numeric($depth)) {
-            throw new LogicException("Option 'depth' must contains an integer value");
+        if ($input->getOption('depth') !== null) {
+            $io->warning('Parameter "depth" is deprecated and has no effect anymore.');
         }
 
         $forceFetch = $input->getOption('force-fetch');
         assert(is_bool($forceFetch));
 
         if (stripos($sql, 'select') === 0 || $forceFetch) {
             $resultSet = $conn->fetchAllAssociative($sql);
+            $io->table(array_keys($resultSet[0] ?? []), $resultSet);
```

Comment:
The current Result API is a bit wrong in that you cannot get column names of an empty result, however, you should. Instead of drawing an empty table (not even sure what it will look like), we might explicitly display something like "Empty result".

Reply:
The empty table will not be drawn at all. But you're right, we could output something nice instead.